webapp/channels/src/utils/message_html_to_component.tsx

```diff
@@ -182,10 +182,10 @@ export function messageHtmlToComponent(html: string, options: Options = {}) {
         processingInstructions.push({
             replaceChildren: true,
             shouldProcessNode: (node: any) => node.attribs && node.attribs[emojiAttrib],
-            processNode: (node: any) => {
+            processNode: (node: any, children: any) => {
                 const emojiName = node.attribs[emojiAttrib];
 
-                return <PostEmoji name={emojiName}/>;
```

Comment:
a little confused here what does children contain in processNode?

Reply:
The children are just whatever the child of this element in the HTML are. In this case, the HTML returned by `renderEmoji` in `utils/emoticons` now passes the matched text (eg. `:carrot:` or `:)`) as the child of this